src/dotnet/APIView/APIViewWeb/Managers/ReviewManager.cs

```diff
@@ -385,6 +389,85 @@ public async Task UpdateReviewBackground(HashSet<string> updateDisabledLanguages
             }            
         }
 
+        public async Task UpdateSwaggerReviewsMetaData()
+        {
+            IList<ReviewModel> reviews = (await _reviewsRepository.GetReviewsAsync(isClosed: false, language: "Swagger", fetchAllPages: true)).ToList();
+            reviews.AddRange(await _reviewsRepository.GetReviewsAsync(isClosed: false, language: "Swagger", fetchAllPages: true));
```

Comment:
Why do we fetch reviews twice? Did you meant to fetch closed reviews in second get reviews?